src/rpc/blockchain.cpp

```diff
@@ -1064,6 +1068,9 @@ UniValue gettxout(const JSONRPCRequest& request)
     }
 
     const CBlockIndex* pindex = LookupBlockIndex(pcoinsTip->GetBestBlock());
+    if (!pindex) {
```

Comment:
Is this even possible? If so then it's new behaviour, should have a test, otherwise should be an assertion.